### ISSUE TITLE
"machine" alias?

### ISSUE THREAD

**Issue by kcgthb** [ChristianKniep/QNIBTerminal#4]
Hi, 

The machine alias/function doesn't seem to be implemented in bashrc:
```
$  git log --pretty=format:'%h' -n 1
13a5797
$ source bashrc
bash: machine: command not found
bash: machine: command not found
bash: machine: command not found
```

**Comment by ChristianKniep** [comment]
Hey there, `docker-machine` is named `machine` on my laptop. It's going to set your PS1 if you source this - using machine to figure out what your active DOCKER server is.

Have you installed docker-machine?

**Comment by kcgthb** [comment]
I had not, thanks for the pointer!

**Comment by ChristianKniep** [comment]
But ain't it stupid from my side to assume `docker-machine` is renamed to `machine`. I might add a test to figure that out.

**Comment by kcgthb** [comment]
Could be useful, yes. :)

Also, I just noticed that you moved to fig to manage containers, and now it looks like fig is deprecated already in favor of Docker Compose.

**Comment by ChristianKniep** [comment]
Yeah... I wanted to switch to docker-compose but some python dependencies broke my fig setup. :)
Is on my list of things - should be pretty painless, since it's the same stuff with a nother name.
You might even use the fig files with docker-compose. Should work, shouldn't it.

I will reopen the issue and add a small test to the bashrc.
